src/app/model/staypoints.ts

```diff
@@ -39,3 +46,36 @@ export function writeStaypointsToGeoJSON(sp: StayPoints, path: string) {
     }
   })
 }
+
+export function writeStaypointClusterArrayToGeoJSON(
+  spcs: StayPointCluster[],
+  path: string
+) {
+  const geojson = {
+    name: 'StayPointClusters',
+    type: 'FeatureCollection',
+    features: [],
+    properties: {
+      trajID: spcs[0].trajID,
+    },
+  }
+  for (const spc of spcs) {
+    geojson.features.push({
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [spc.coordinates[1], spc.coordinates[0]],
```

Comment:
Same here: What happens, if `spc.coordinates.length === 0`?

Reply:
A staypoint cluster is required to have exactly two coordinates, so I think this is fine.